scripts/gen.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -e
+
+if (( $# < 2 )); then
+    echo "Go wrappers generator script. Make sure to specify the following params:"
+    echo " - first parameter: the contracts source directory"
+    echo " - second parameter: the contracts names (including the .sol extension) separated by a space"
+    echo "the output files will be in the ./wrappers directory."
+    exit 1
+fi
+
+forge build > /dev/null
+
+cd "$1"
+
+for file in "${@: 2}"; do
+    mkdir -p ../wrappers/"${file}"
+    contractName=$(basename "${file}" .sol)
+
+    jq .abi < ../out/"${file}"/"${contractName}".json > ../out/"${file}"/"${contractName}".abi
+    jq .bytecode.object < ../out/"${file}"/"${contractName}".json | cut -d \" -f 2 > ../out/"${file}"/"${contractName}".bin
```

Comment:
What is the `cut -d \" -f 2` doing here? If it's manipulating JSON then would using `jq` be better?

Reply:
[c567737](https://github.com/celestiaorg/quantum-gravity-bridge/pull/139/commits/c567737e2c6aef97f1d7d18636a09f841248eeae)